contracts/cryptoblades.sol

```diff
@@ -433,6 +433,20 @@ contract CryptoBlades is Initializable, AccessControlUpgradeable {
         }
     }
 
+    function mintWeaponN(uint32 num)
+        public
+        onlyNonContract
+        oncePerBlock(msg.sender)
+        requestPayFromPlayer(mintWeaponFee * num)
+    {
+        require(num > 0);
```

Comment:
Better be safe than sorry and just enforce a maximum value as well. Can be changed easily later on anyway.